site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Get and apply a remote composite package"
+linkTitle: "Composite Package"
+weight: 7
+type: docs
+description: >
+  Fetch a composite package from a remote git repository and apply its contents to
+  a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Composite packages feature support is in alpha phase
+
+{{% /pageinfo %}}
+
+This guide walks you through an example to get, view, set and apply contents of a
+kpt package which has nested subpackages in it. A kpt package is a directory of
```

Comment:
Is "nested subpackage" redundant? Can there be a subpackage that's not "nested"?

There are few terms being thrown around which you need to define:

- Composite
- Subpackage
- Nested

Eliminate terms that are redundant.

Reply:
Removed `nested` as `recursive` is good enough. Defined the other two terms.